contrastsecurity/datadog_checks/contrastsecurity/conf.yaml.example

```diff
@@ -1,20 +0,0 @@
-init_config:
-
-instances:
-
-## Log Section
```

Comment:
> Just contrastsecurity.d/conf.yaml file alone will allow agent to use the conf file for logs

@AlexandreYang we don't need to keep this default example?

Reply:
The issue is that, for `extras`, for this file to be copied, we need to created a wheel and then use `agent integration install` command. Which is not possible since there is no python integration anymore.